tools/docker/tests/test_flows.sh

```diff
@@ -85,21 +166,20 @@ main() {
         case "$1" in
             -v | --verbose)     VERBOSE=true; shift ;;
             -h | --help)        usage; exit 0; shift ;;
-            -a | --all)         TEST_ALL=true; shift ;;
-            --test-topology)    TEST_TOPOLOGY=true; shift;;
             -- ) shift; break ;;
             * ) err "unsupported argument $1"; usage; exit 1 ;;
         esac
     done
-    
-
+    [ -z "$@" ] && set ${ALL_TESTS}
+    echo $@
     test_init
-    [ -n "$TEST_ALL" -o -n "$TEST_TOPOLOGY" ] && report "test topology query" test_topology $AL_MAC
-    # TODO add more tests here    
+    for test in $@; do
+       report "test_${test}" test_${test}
+    done
 }
 
+ALL_TESTS="topology initial_ap_config ap_config_renew ap_config_bss_tear_down channel_selection ap_capability_info_reporting client_capability_info_reporting link_metric_collection steering_mandate_opportunity client_steering client_association backhaul_optimization layer_data_payload_trigger layer_data_payload"
```

Comment:
Two minor nits here:

- I find it more logical to put the defaults that may be overridden by command-line options here, but to put 'global const' variables in the beginning of the script (i.e. before defining functions, i.e. the place where you'd define global variables in C code). But this is just personal preference.
- The long line should be split. Since it's between quotes and since it will be whitespace-expanded where it is used, you can just split it anywhere, so:

```
ALL_TESTS="topology initial_ap_config ap_config_renew ap_config_bss_tear_down
    channel_selection ap_capability_info_reporting client_capability_info_reporting
    ..."
```

(I didn't count the characters here, but we wrap at 100 columns, cfr. .clang-format.)